content/status.mdx

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Could you add this page to the `nav.yml` as well?

Reply:
That was another question 😅 
>I haven't added the /status to the navigation yet. I wasn't really sure where to place it best as we don't seem to have a nav on the primer.style

Where should it be on the nav? At the top and a standalone link?

Reply:
Have a `status` section on the nav and put it under